### PR TITLE
HW7 Practiced concurrency package

### PR DESCRIPTION
**Pull request by taras309**

### CODE REVIEW

File: src/com/concurency/singleton/Singleton.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        try {
+            Thread.sleep(1_000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        System.out.println("Instance count is: " + instanceCount);
+    }
+
+    synchronized public static Singleton getInstance() {
```
Comment (Taniann) [inline src/com/concurency/singleton/Singleton.java]: better firstly access modificator

File: src/com/concurency/singleton/Singleton.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if (instance == null) {
+            instance = new Singleton();
+            instanceCount++;
```
Comment (Taniann) [inline src/com/concurency/singleton/Singleton.java]: why do you need instance cout if singleton must have just one instance

Comment (Taniann) [reply src/com/concurency/singleton/Singleton.java]: and try to make singleton with double checking

File: src/com/concurency/singleton/Singleton.java (hunk 1 above)
Comment (taras309) [reply src/com/concurency/singleton/Singleton.java]: Done

File: src/com/concurency/singleton/Singleton.java (hunk 2 above)
Comment (taras309) [reply src/com/concurency/singleton/Singleton.java]: Done

**Pull request merged**